assets/js/modules/analytics/components/settings/SettingsView.js

```diff
@@ -95,6 +100,23 @@ export default function SettingsView() {
 						<DisplaySetting value={ profileID } />
 					</p>
 				</div>
+				<div className="googlesitekit-settings-module__meta-item">
+					<h5 className="googlesitekit-settings-module__meta-item-type">
+						&nbsp;
+					</h5>
+					<Link
+						href={ editViewSettingsURL }
+						external
+						className="googlesitekit-settings-module__inline-link"
+					>
+						{ createInterpolateElement(
+							__( 'Edit <VisuallyHidden>Universal Analytics property view </VisuallyHidden>in Analytics', 'google-site-kit' ),
```

Comment:
Let's use two strings for the variants instead – this will be simpler for translators as well.
The screen reader variant can simply be set as the `aria-label` of the `Link`.

Reply:
This is copied from before in the code.  I'll comment there now @aaemnnosttv